.testcoverage.yml

```diff
@@ -23,7 +23,7 @@ override:
     path: ^internal/istiostatus$
   - threshold: 87
     path: ^internal/otelcollector/config/otlpexporter$
-  - threshold: 77
+  - threshold: 73
```

Comment:
Because of the functional options I've implemented. Didn't we agree to increase it back up in my next refactoring PR? Since the functional options might not be needed anymore by then.

Reply:
please don't forget to revert this change when you remove the reason for it